src/main/java/com/project/momo/security/oauth/GithubOAuthAttributes.java

```diff
@@ -12,7 +12,9 @@
     private static final String USER_NAME_ATTRIBUTE_NAME = "id";
 
     public GithubOAuthAttributes(String oAuthId, String name, String email, String imageUrl) {
-        super(OauthType.GITHUB, oAuthId, name, email, imageUrl);
+        super(OauthType.GITHUB.getValue(), oAuthId, name, email, imageUrl);
+        System.out.println(OauthType.GITHUB.getValue());
+
```

Comment:
```suggestion
        super(OauthType.GITHUB.getValue(), oAuthId, name, email, imageUrl);
```
의미없는 공백 + sout은 제거해주세요

Reply:
넵 주의하겠습니다.